wifiphisher/deauth.py

```diff
@@ -7,8 +7,8 @@
 
 import threading
 import scapy.layers.dot11 as dot11
-from scapy.all import *
-from constants import *
+import scapy.arch.linux as linux
+import wifiphisher.constants as constants
```

Comment:
I understand the problem with relative imports. However, i want to be able to call a constant just by typing its name (instead of `constants.CONSTANT_NAME`). How about `from wifiphisher.constants import *`?